lib/src/remote_entity_actions.dart

```diff
@@ -200,6 +200,22 @@ class FailUpdateOne<T> {
   Map<String, dynamic> toJson() => {'entity': entity, 'error': error};
 }
 
+class FailUpdateOneByIdWith<T, R> {
+  const FailUpdateOneByIdWith({
+    required this.id,
+    required this.entity,
```

Comment:
You don't really need the `With` here, unless you think we'll be doing different handling based on the type used for the update?

`FailUpdateOneById<T>` is probably enough?